data-generation/src/main/resources/benchmarks/1651588266-LaTeX-test.csv

```diff
@@ -0,0 +1,11 @@
+function,value,timeunit,number of runs
```

Comment:
Er det ønskelig å commitee alle disse .csv filene? Du kan kjøre noe moro .gitignore sikkert for å commite alle csver som inneholder et kodeord eller noe.

Reply:
Oii, hahhaa, de dukket plutselig opp her ja 🤦